  <launch>

  <node pkg="vel_safety_limiter" type="vel_limiter" name="vel_limiter" output="screen">
    <param name="base_frame" type="string" value="base_link" />
    <param name="high_distance_threshold" value="0.6"/>
    <param name="low_distance_threshold" value="0.2"/>
    <param name="vel_topic" type="string" value="/ubiquity_velocity_controller/cmld_vel" />
    <remap from="cmd_vel" to="/cmd_vel/limit" />
     <remap from="scan" to="/scan_filtered" />
  </node> 

  </launch>